src/logic/settings/reducer/advanced.ts

```diff
@@ -0,0 +1,25 @@
+import { Action, handleActions } from 'redux-actions'
```

Comment:
I think this is a lot of boilerplate for just one setting. Perhaps we could move this over to what is currently the 'appearance' reducer and aim to migrate that to being 'settings'. What do you think?

Alternatively, we could use the cached state but it does seem better suited to the store.